### PR TITLE
End-to-end build

### PR DESCRIPTION
**Pull request by anpaz-msft**

### CODE REVIEW

**Comment by rmshaffer** [comment]
FYI, the pytest failures in your CI run are due to issue https://github.com/microsoft/iqsharp/issues/310, which I'm actively working on (I believe it requires a fix in jupyter-core).

File: build/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - minimal
+  - normal
+  - full
+
+trigger: none
+
+pr:
+- main
+- feature/*
+- features/*
```
Comment (rmshaffer) [inline build/e2e.yml:18]: Just curious, why does this one contains `features/*` but `ci.yml` doesn't?

File: build/ci.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 name: $(Build.Major).$(Build.Minor).$(date:yyMM).$(DayOfMonth)$(rev:rr)
-trigger:
-- master
+
+trigger: none
```
Comment (rmshaffer) [inline build/ci.yml:3]: Why not trigger on commits to `main` here? Will that happen somewhere else?

**Review by rmshaffer — DISMISSED** [review]

File: build/ci.yml (hunk 2 above)
Comment (anpaz-msft) [reply build/ci.yml:3]: There is not much value to trigger another build on master, and we have limited number of agents for public projects.
Release builds on push to master are indeed triggered on a different build.

**Review by rmshaffer — APPROVED** [review]

**Pull request merged**